content/applications/websites/ecommerce/products.rst

```diff
@@ -4,275 +4,402 @@
+field blank and toggle the :guilabel:`Show on product page` switch.
 
-.. image:: products/products-digital-files.png
-   :align: center
-   :alt: Digital Files menu
+.. image:: products/digital-files.png
+   :alt: digital file available before payment on the  product page
 
-For the configuration:
+.. _ecommerce-products-digital-files-after-payment:
 
-- :guilabel:`Name`: the name of your file;
-- :guilabel:`Type:` select if it is either a **file** or a **URL**. Accordingly, you either have a
-  :guilabel:`File Content (base64)` field to upload your file, or a :guilabel:`URL` field to enter
-  your URL.
-- :guilabel:`Website`: the website on which that file is *available*. If you want it available for
-  *all* websites, leave it empty.
+Digital files available after payment
+-------------------------------------
 
-The file is then available after checkout in the :guilabel:`Sales Orders` section, found on the
-customer's portal.
+To make your file available (after payment), set the :guilabel:`Visibility` field to
+:guilabel:`Confirmed order` and turn off the :guilabel:`Show on product page` switch.
 
-Product configuration
-=====================
+.. _ecommerce-products-translation:
 
-Multiple languages
-------------------
+Translation
+===========
 
-If multiple languages are available on your website and you wish to have the product's information
-translated, it is necessary to encode this translated information in the **product's template**.
-Fields with multiple languages available are identifiable by their abbreviation language (ex.
-:abbr:`EN (English)`) next to their field.
+If multiple languages are available on your website, you can translate a product's information
+directly on the :ref:`product form <ecommerce-products-product-form>`. Fields that support multiple languages
+are identifiable by their abbreviation language (e.g., EN) next to their field.
 
 .. image:: products/products-field-translation.png
-   :align: center
    :alt: Field translation
 
-The **eCommerce-related** fields to translate are:
+The eCommerce-related fields to translate are:
 
 - :guilabel:`Product name`;
 - :guilabel:`Out-of-Stock Message` (under the :guilabel:`Sales` tab);
-- :guilabel:`Sales Description` (under the :guilabel:`Sales` tab);
+- :guilabel:`Sales Description` (under the :guilabel:`Sales` tab).
 
 .. note::
-   Having untranslated content on a web page may be detrimental to the user experience and
-   :doc:`SEO <../../websites/website/pages/seo>`.
-.. note::
-   To check the language(s) of your website, go to :menuselection:`Website --> Configuration -->
-   Settings --> Website Info section`.
+   - Having untranslated content on a web page may be detrimental to the user experience and
+     :doc:`SEO <../../websites/website/pages/seo>`. You can use the
+     :doc:`Translate <../website/configuration/translate>` feature to allow automatic translation of
+     standard terms and provide a tool for manual content translation.
```

Comment:
```suggestion
   - Having untranslated content on a web page may be detrimental to the user experience and
     :doc:`SEO <../../websites/website/pages/seo>`. You can use the
     :doc:`Translate <../website/configuration/translate>` feature to translate the page's content
```
- The last sentence sounds a bit too vague (what are *standard terms*?)
- The transition between the two parts of the last sentence (allow automatic translation xxx and provide a tool for xxx) is a bit clumsy and doesn't really flow

I tried to rephrase the sentence but actually, I think we can just simplify it since the Translations page covers the rest.